tools/rimage/config/lnl.toml

```diff
@@ -56,7 +56,7 @@ name = "ADSPFW"
 load_offset = "0x40000"
 
 [module]
-count = 27
+count = 28
```

Comment:
Would be good to upgrade lnl.toml also to use "__COUNTER__" like mtl already is. But not a blocker for this PR, just want to log a note.

Reply:
verified here: https://github.com/abonislawski/sof/commit/8289739f82061a79d92d5552872cffcf97c849e2
but not sure if this should go upstream since module specific toml files contains MTL-only valid data (CPC) but on the other hand this data is outdated and anyway will work on MTL and LNL